### PR TITLE
feat: Wallet Select Modal Onboarding

### PR DESCRIPTION
**Pull request by JoseRFelix**
## What is the purpose of the change

Adds a new onboarding modal for Wallet Select. 

### ClickUp Task

[ClickUp Task URL](https://app.clickup.com/t/866ab6ea1)

## Key Workflows to test

- Connect to any wallet
- Connect to Keplr Mobile
- Be able to get Keplr mobile download link QR

### CODE REVIEW

File: packages/web/modals/wallet-select.tsx [hunk 1]
```diff
@@ -523,51 +526,110 @@ const RightModalContent: FunctionComponent<
...
+                  width={186}
+                  height={186}
+                />
+              </div>
+
+              <div className="flex max-w-sm flex-col gap-3">
+                <h1 className="subtitle1">Your keys, your funds.</h1>
+                <p className="body2 text-osmoverse-300">
+                  The beauty of a decentralized exchange. None of your assets
+                  are ever with us.
```
Comment (jonator) [inline packages/web/modals/wallet-select.tsx:592]: Needs to be translated.

File: packages/web/modals/wallet-select.tsx [hunk 2]
```diff
@@ -605,21 +660,33 @@ const QRCodeView: FunctionComponent<{ wallet?: ChainWalletBase }> = ({
...
+      <div className="mb-7 flex flex-col items-center justify-center gap-3">
+        <p className="flex items-center gap-2 rounded-2xl bg-osmoverse-900 px-10 py-3 text-osmoverse-200">
+          <span>Tap the</span>
+          <Image src="/icons/scan.png" alt="scan icon" width={28} height={28} />
+          <span>button</span>
+        </p>
+
+        <p className="body2 max-w-sm text-center text-wosmongton-100">
+          on the top right corner of your Keplr mobile app. Scan the QR code
+          below to connect.
```
Comment (jonator) [inline packages/web/modals/wallet-select.tsx:676]: Also translated

File: packages/web/modals/wallet-select.tsx [hunk 3]
```diff
@@ -110,129 +143,277 @@ export const WalletSelectModal: FunctionComponent<
...
+      wallet?: ChainWalletBase | (typeof WalletRegistry)[number]
+    ) => void;
+  }
+> = observer(({ walletRepo, onConnect }) => {
+  const { isMobile } = useWindowSize();
+  const t = useTranslation();
 
-    const [lazyWalletInfo, setLazyWalletInfo] =
-      useState<(typeof WalletRegistry)[number]>();
+  const wallets = [...WalletRegistry]
```
Comment (jonator) [inline packages/web/modals/wallet-select.tsx:277]: Should this be a useMemo? Or is this component not expected to render much?

File: packages/web/modals/wallet-select.tsx [hunk 4]
```diff
@@ -110,129 +143,277 @@ export const WalletSelectModal: FunctionComponent<
...
-          })
-          .catch(handleConnectError);
-      } else {
-        installedWallet
-          ?.connect(sync)
-          .then(() => {
-            onConnectProp?.();
-          })
-          .catch(handleConnectError);
+  const categories = wallets.reduce(
```
Comment (jonator) [inline packages/web/modals/wallet-select.tsx:323]: useMemo here as well

**Review by jonator — CHANGES_REQUESTED** [review]

**Review by jonator — CHANGES_REQUESTED** [review]
Can we add arrow buttons here, like in instagram flow?

The dots are hard to use to navigate, but are nice to jump directly to slides.
Screenshot 2023-07-29 at 7 09 23 AM

Nit, but since we're editing this area:

Can we match the wallet icon with the OSMO icon? Looks like it's osmoverse 300.
Screenshot 2023-07-29 at 7 10 23 AM

Bug: when I disconnected Keplr then clicked Keplr Mobile (to see the QR code) I saw this error appear behind the modal:
Screenshot 2023-07-29 at 7 18 56 AM

**Comment by JoseRFelix** [comment]
@jonator Thanks for the feedback and review! I have addressed it accordingly. 

> Bug: when I disconnected Keplr then clicked Keplr Mobile (to see the QR code) I saw this error appear behind the modal:

This happens because the `NEXT_PUBLIC_WALLETCONNECT_PROJECT_KEY` is not set on the `.env` file

**Review by jonator — APPROVED** [review]

**Comment by jonator** [comment]
Great work!

**Pull request merged**